app/routes/questionnaire.py

```diff
@@ -95,11 +95,12 @@ def before_questionnaire_request():
         "questionnaire request", method=request.method, url_path=request.full_path
     )
 
-    handle_language()
+    handle_language(metadata)
 
-    session_store = get_session_store()
     # pylint: disable=assigning-non-slot
```

Comment:
Yeah this is still needed

Reply:
Require for the method `g.schema = load_schema_from_metadata(
        metadata=metadata, language_code=get_locale().language
    )`

Otherwise get pylint error: `Assigning to attribute 'schema' not defined in class slots (assigning-non-slot)`